### PR TITLE
Add admin registration

### LINKED ISSUE #72: Add admin register functionality

### PR DESCRIPTION
**Pull request by maksym-strus**
Added admin registration (allowed only for super admin). Closes #72

### CODE REVIEW

File: src/configs/index.js [hunk 1]
```diff
@@ -145,8 +164,17 @@ export const config = {
     INVALID_EMAIL_MESSAGE: 'Некоректна email адреса',
     ENTER_EMAIL_MESSAGE: 'Введіть email',
     PASSWORD_MIN_LENGTH_MESSAGE: 'Пароль повинен містити не менше 8 символів',
+    PASSWORD_MAX_LENGTH_MESSAGE: 'Пароль повинен містити не більше 20 символів',
     PASSWORD_LANG_MESSAGE: 'Використовуйте латиницю різних регістрів та цифри',
-    ENTER_PASSWORD_MESSAGE: 'Введіть пароль'
+    ENTER_FIRSTNAME_MESSAGE: "Введіть ім'я",
```
Comment (Alan18081) [inline src/configs/index.js:169]: Please, use single quotes

File: src/configs/index.js [hunk 2]
```diff
@@ -145,8 +164,17 @@ export const config = {
...
+    ENTER_LASTNAME_MESSAGE: 'Введіть прізвище',
+    ENTER_PASSWORD_MESSAGE: 'Введіть пароль',
+    FIRSTNAME_MIN_LENGTH_MESSAGE: "Ім'я повинно містити не менше 2 символів",
```
Comment (Alan18081) [inline src/configs/index.js:172]: Please, use single quotes

File: src/configs/index.js [hunk 3]
```diff
@@ -145,8 +164,17 @@ export const config = {
...
+    LASTNAME_MIN_LENGTH_MESSAGE: 'Прізвище повинно містити не менше 2 символів',
+    FIRSTNAME_MAX_LENGTH_MESSAGE: "Ім'я повинно містити не більше 30 символів",
```
Comment (Alan18081) [inline src/configs/index.js:174]: Please, use single quotes

File: src/services/local-storage.service.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 export const clearLocalStorage = () => {
   const horondi = {
-    HORONDI_AUTH_TOKEN: null,
+    HORONDI_AUTH_TOKEN: '',
```
Comment (Alan18081) [inline src/services/local-storage.service.js:3]: Revert this change, it is better to have `null` as an empty value

Comment (maksym-strus) [reply src/services/local-storage.service.js:3]: it helps to work with routes where we don`t use tokens. {token: null} works wrong

File: src/configs/index.js (hunk 3 above)
Comment (maksym-strus) [reply src/configs/index.js:174]: fixed

File: src/configs/index.js (hunk 2 above)
Comment (maksym-strus) [reply src/configs/index.js:172]: fixed

File: src/configs/index.js (hunk 1 above)
Comment (maksym-strus) [reply src/configs/index.js:169]: fixed
